.quickstart/quickstart.yml

```
@@ -0,0 +1,17 @@
+database_key: amazon_selling_partner_database
+schema_key: amazon_selling_partner_schema
+
+dbt_versions: ">=1.3.0 <2.0.0"
+
+table_variables:
+  amazon_ads__portfolio_history_enabled:
```

Is this meant to be the following instead of an amazon ads variable?
```suggestion
  amazon_selling_partner__using_orders_module:
```

---

updated